client/my-sites/site-monitoring/components/site-monitoring-bar-chart/use-metrics-bar-chart-data.ts

```diff
@@ -0,0 +1,74 @@
+import { useMemo } from 'react';
+import { TimeRange } from '../../main';
+import { PeriodData, useSiteMetricsQuery } from '../../use-metrics-query';
+
+const STATUS_CODES = [ 200, 401, 400, 404, 500 ];
```

Comment:
I guess we need to figure out which status codes we want to show. Showing all of them probably isn't realistic.

Reply:
Yes, I had to decide between
Dynamically extract the status codes returned by the API, which in my dummy sites were just three `200`, `404`, and '400` and in the chart, status codes with 0 will not appear.

Versus

Displaying "selected" status codes. I took this path to make the chart look like the design.